# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/faad2/gcc40.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

This fixes the error:

mp4ff.c:91: error: static declaration of 'mp4ff_track_add' follows non-static declaration
mp4ffint.h:304: error: previous declaration of 'mp4ff_track_add' was here

diff -dur faad2/common/mp4ff/mp4ff.c src.faad2.1129998164.4669.1585831089/faad2/common/mp4ff/mp4ff.c
--- faad2/common/mp4ff/mp4ff.c	2004-01-11 16:52:18.000000000 +0100
+++ src.faad2.1129998164.4669.1585831089/faad2/common/mp4ff/mp4ff.c	2005-10-23 22:28:23.000000000 +0200
@@ -87,7 +87,7 @@
     if (ff) free(ff);
 }
 
-static void mp4ff_track_add(mp4ff_t *f)
+void mp4ff_track_add(mp4ff_t *f)
 {
     f->total_tracks++;
 
